server/planning/events/events.py

```diff
@@ -223,7 +223,7 @@ def on_create(self, docs):
                 recurring_events = generate_recurring_events(event)
                 generated_events.extend(recurring_events)
                 # remove the event that contains the recurring rule. We don't need it anymore
-                docs.remove(event)
+                docs.remove(event)  # todo: why we remove that event and not update it?
```

Comment:
I think we can fix this by skipping the first event inside `generate_recurring_events` function, that way the passed in Event will not need to be removed.

Reply:
I see, will avoid further changes at this stage but might fix it later